src/upload-tests.proj

```diff
@@ -102,10 +102,21 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyTestILCToTestHostDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
```

Comment:
TestILC is a big package(~270MB), so I thought that doing it on local builds would slow things down plus it can be run out from a share since we don't have to mklink to any of its contents, so It might be useful to just have it in a share and use that one everytime as opposed to have to recopy everytime that you nuke your bin folder.